Settings.h

```diff
@@ -78,4 +79,6 @@ Settings* Settings_new(int initialCpuCount);
 
 void Settings_invertSortOrder(Settings* this);
 
+const Settings* Settings_getSettings(void);
+
```

Comment:
This is already handled differently in the rest of the code. Cf. first field in `ProcessList` structure ;-)